orchprovider/k8s/v1/util.go

```diff
@@ -193,6 +193,21 @@ func (p *VolumeMarkerBuilder) AddControllerStatuses(pod k8sApiV1.Pod) {
 	_ = p.AddMultiples(string(v1.JivaControllerStatusVK), status, true)
 }
 
+func (p *VolumeMarkerBuilder) AddControllerContainerStatuses(cp k8sApiV1.Pod) {
```

Comment:
It will be good to pass Deployment. It will have n no of Pods. Each Pod will have m number of Containers.